uint32              current_organism
uint32              generation_number
uint32              population_size
float64             population_fitness
float64             population_diversity
genetic_algorithm/Organism[]    organisms
